db/query_test.go

```diff
@@ -55,9 +51,9 @@ func TestQueryChannelsStatsView(t *testing.T) {
 	closeErr := results.Close()
 	assert.NoError(t, closeErr, "Close error")
 
-	channelQueryCountAfter := base.ExpvarVar2Int(db.DbStats.StatsGsiViews().Get(queryCountExpvar))
-	channelQueryTimeAfter := base.ExpvarVar2Int(db.DbStats.StatsGsiViews().Get(queryTimeExpvar))
-	channelQueryErrorCountAfter := base.ExpvarVar2Int(db.DbStats.StatsGsiViews().Get(errorCountExpvar))
+	channelQueryCountAfter := db.DbStats.NewStats.GSIStats(fmt.Sprintf("%s.%s", DesignDocSyncGateway(), ViewChannels)).QueryCount.Value.(int)
```

Comment:
If QueryTypeChannels is used for GSI, it's probably reasonable to create constants or vars for the view equivalents, instead of building them with fmt.Sprintf every time.

Reply:
I've added a const to use for the formatting bit: "%s.%s" not sure its nice to have the full thing as a const incase names change.